src/components/screens/ScoreSetEditor.vue

```diff
@@ -1098,7 +1098,6 @@ export default {
     searchTargetGeneIdentifiers: function (dbName, event) {
       const searchText = (event.query || '').trim()
       if (searchText.length > 0) {
-        this.targetGene.externalIdentifiers[dbName].identifier = null
```

Comment:
just making sure this is intentional as it seems different from the other changes

Reply:
Yeah it was. With that line in there any time a user types the text box gets cleared and the text will only reappear once the Autocomplete loses its focus. I deleted it so the text never gets hidden in the text box, but there isn't any change in functionality.